### PR TITLE
Add content/pt-br/blue_green_deployment.md

### PR DESCRIPTION
**Pull request by jessicalins**
Reopening - same content as https://github.com/cncf/glossary/pull/274, but now with an organized tree/commits. Sorry for the confusion :sweat_smile: 

Signed-off-by: Jéssica Lins <jessicaalins@gmail.com>

### CODE REVIEW

**Comment by jessicalins** [comment]
cc @brunoguidone

File: content/pt-br/blue_green_deployment.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Blue Green Deployment
+status: Completed
+category: Conceito
```
Comment (edsoncelio) [inline content/pt-br/blue_green_deployment.md:4]: Conceito to conceito

**Review by edsoncelio — CHANGES_REQUESTED** [review]
Hi @jessicalins, i just left a small comment :)

**Review by brunoguidone — APPROVED** [review]
Review ok!

**Pull request merged**